### PR TITLE
Added thumbnails to insert preview.

### LINKED ISSUE #618: thumbnails not showing upon adding a step
### Please describe the problem (or idea)

> What happened just before the problem occurred? Or what problem could this idea solve?

 on clicking +Add button, the toolbar appears but without thumbnails

screenshot 2019-01-06 at 21 04 12

### Please show us where to look

http://sequencer.publiclab.org...

### Browser, version, and operating system
Chrome, MacOS

> Many bugs are related to these -- please help us track it down and reproduce what you're seeing!

****

## Thank you!

Your help makes Public Lab better! We *deeply* appreciate your helping refine and improve this site.

### PR DESCRIPTION
**Pull request by Divy123**
Fixes #618
Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

Thanks!

### CODE REVIEW

**Comment by Divy123** [comment]
ezgif com-video-to-gif

**Comment by Divy123** [comment]
@jywarren please review.
Sorry for closing the last PR but there were too many merge conflicts in the dist files and it was getting difficult to resolve.

**Comment by jywarren** [comment]
Oh this is great! Would it be possible to center the pane?

File: examples/lib/intermediateHtmlStepUi.js [hunk 1]
```diff
@@ -50,12 +50,64 @@ function IntermediateHtmlStepUi(_sequencer, step, options) {
         </section>\
         </div>';
   }
+
+  function generatePreview(previewStepName, customValues, path) {
```
Comment (jywarren) [inline examples/lib/intermediateHtmlStepUi.js]: Also, how redundant is this with the original code? Would it be possible to have them share significant code? Thanks!

Comment (Divy123) [reply examples/lib/intermediateHtmlStepUi.js]: Sure @jywarren Will do that.

File: dist/image-sequencer-ui.js [hunk 2]
```diff
@@ -489,6 +541,13 @@ function IntermediateHtmlStepUi(_sequencer, step, options) {
       .insertAdjacentElement('afterend',
         addStepUI
       );
+      console.log(sequencer);
```
Comment (jywarren) [inline dist/image-sequencer-ui.js]: Oops, looks like a stray log file and some commented out code here?

Comment (jywarren) [reply dist/image-sequencer-ui.js]: Also oops i commented on the /dist/ version... 😂 anyways you get it!

File: examples/demo.js [hunk 3]
```diff
@@ -213,10 +183,12 @@ window.onload = function() {
     onLoad: function onFileReaderLoad(progress) {
       var reader = progress.target;
       var step = sequencer.images.image1.steps[0];
+      var util=IntermediateHtmlStepUi(sequencer);
       step.output.src = reader.result;
       sequencer.run({ index: 0 });
       step.options.step.imgElement.src = reader.result;
-      updatePreviews(reader.result);
+      updatePreviews(reader.result,'addStep');
```
Comment (jywarren) [inline examples/demo.js]: Would this only run if there are some shown? And just checking, does this code all run on the final output or the original input? Maybe it ought to run on the final output; the last step before the add interface... what do you think?

File: examples/lib/insertPreview.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
+function generatePreview(previewStepName, customValues, path, selector) {
```
Comment (jywarren) [inline examples/lib/insertPreview.js]: This looks great!

File: examples/lib/intermediateHtmlStepUi.js [hunk 5]
```diff
@@ -65,6 +67,7 @@ function IntermediateHtmlStepUi(_sequencer, step, options) {
       .insertAdjacentElement('afterend',
         addStepUI
       );
+      updatePreviews(_sequencer.images.image1.steps[0].options.step.imgElement.src,'insertStep');
```
Comment (jywarren) [inline examples/lib/intermediateHtmlStepUi.js]: Wow this is long! Does insertStep not have access to the current step somehow simpler?

File: dist/image-sequencer-ui.js (hunk 2 above)
Comment (Divy123) [reply dist/image-sequencer-ui.js]: Sure! :smile:

File: examples/lib/intermediateHtmlStepUi.js (hunk 5 above)
Comment (Divy123) [reply examples/lib/intermediateHtmlStepUi.js]: Please review.

**Comment by jywarren** [comment]
This is ready to merge as soon as it's rebased - and if grunt build is run again the changes will also reflect in dist/image-sequencer-ui.js. - thanks!!!!

**Comment by Divy123** [comment]
@publiclab/reviewers please review.

**Comment by jywarren** [comment]
Awesome!!! Thanks!!

**Pull request merged**